Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:10.12.0
+
+ARG HudlCiToolsTagName
+ARG npmRegistryUrl
+ARG teamcityBuildBranch
+ARG buildNumber
+ARG npmPackageDir
+ARG teamCityNpmPassword
+ARG teamCityNpmAuthToken
+
+RUN npm i -g hudl-ci-tools@${HudlCiToolsTagName} --registry=${npmRegistryUrl}
+
+# install yarn 1.11.1
+RUN curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.11.1
```

Comment:
There's also a `1.12.3` version of yarn that could be worth updating to. Unfortunately the `node:10.13.0` image did not update the yarn version used 😑 so this manual update is still required.